### PR TITLE
Implement repr for History object

### LINKED ISSUE #2: Give History a repr that looks like a dict of most recent vales.
@ericdill

### PR DESCRIPTION
**Pull request by ericdill**
Closes #2

### CODE REVIEW

File: history.py [hunk 1]
```diff
@@ -66,7 +66,9 @@ def __init__(self, fname):
         self._keys = self.past(self.RESERVED_KEY_KEY)
         for k in self._keys:
             self._cache[k] = self.past(k)
-
```
Comment (danielballan) [inline history.py:69]: Is there a blank line missing here?

File: test_history.py [hunk 2]
```diff
@@ -114,3 +115,13 @@ def test_protected_key():
...
+
+
+def test_repr():
+    h.clear()
+    kvpairs = (('foo', 'bar'), ('spam', 'spam spam spam'))
+    dct = {}
+    for k, v in kvpairs:
+        dct[k] = v
+        h[k] = v
+    assert repr(h) == repr(dct)
```
Comment (danielballan) [inline test_history.py]: FYI, `assert_equal` gives a better error message in the event of failure, but probably not worth going back for that.

Comment (ericdill) [inline test_history.py]: Ah, right. Good catch!  

I've been working with py.test for pyqtgraph which does give helpful error messages with just `assert` and that leaked over into this

**Comment by danielballan** [comment]
Sweet. Looks good.

**Pull request merged**